angular.json

```diff
@@ -108,7 +108,7 @@
                 },
                 "fonts": false
               },
-              "sourceMap": false,
+              "sourceMap": true,
```

Comment:
This setting disabled the ability to debug! 
Can move into a separate branch if necessary